pkgs/by-name/li/likwid/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchurl
+, perl
+, coreutils
+, gnugrep
+}:
+
+stdenv.mkDerivation rec {
+  pname = "likwid";
+  version = "5.3.0";
+
+  src = fetchurl {
+    url = "https://ftp.fau.de/pub/likwid/likwid-${version}.tar.gz";
+    hash = "sha256-wpDlVMQlMSSsKriwVuFO5NI5ZrjJ+/oQuoH3WuVDzk4=";
+  };
+
+  nativeBuildInputs = [ perl ];
+
+  hardeningDisable = [ "format" ];
+
+  preBuild = ''
+    patchShebangs bench/ perl/
+    substituteInPlace Makefile --replace 'install -m 4755 $(INSTALL_CHOWN)' 'install -m 755'
+    substituteInPlace src/access-daemon/accessDaemon.c \
+      --replace 'cat /proc/cpuinfo | grep \"processor\" | sort -u | wc -l' '${coreutils}/bin/nproc --all' \
+      --replace 'cat /proc/cpuinfo | grep \"physical id\" | sort -u | wc -l' \
+        '${coreutils}/bin/cat /proc/cpuinfo | ${gnugrep}/bin/grep \"physical id\" | ${coreutils}/bin/sort -u | ${coreutils}/bin/wc -l'
+  '';
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with lib; {
+    homepage = "https://hpc.fau.de/research/tools/likwid/";
+    description = "LIKWID Performance Tools";
```

From https://github.com/RRZE-HPC/likwid

```suggestion
    description = "Performance monitoring and benchmarking suite ";
```

No self-reference on short description.